src/desktop/apps/artist/client.tsx

```diff
@@ -17,7 +17,7 @@ buildClientApp({
       sd.ARTIST_COLLECTIONS_RAIL_QA === "experiment" &&
       sd.CURRENT_USER &&
       sd.CURRENT_USER.type === "Admin", // TODO: update after Artist Collections Rail a/b test
-  },
+  } as any,
```

Comment:
@alloy - curious what the best approach is for adding additional properties here and extending `SystemContextProps['context']`?

Reply:
Currently I’m thinking that perhaps each app should export its own `buildServerApp` and `buildClientApp` functions that can be augmented with their specific extended context interface, but let’s follow-up on that so these PRs don’t go stale.